tools/downloader/README.md

```diff
@@ -255,7 +255,8 @@ If the specified precision is not supported for a model, that model will be skip
 
 The script will attempt to locate Model Optimizer using the environment
 variables set by the OpenVINO&trade; toolkit's `setupvars.sh`/`setupvars.bat`
-script. You can override this heuristic with the `--mo` option:
+script. Or trying to use PyPI distribution.
```

Comment:
> Or trying to use PyPI distribution.

That is not a complete sentence. It should be combined the preceding one.

But besides that, it also needs reworking. First, it doesn't actually fit the previous sentence. If you combine them, you get "The script will attempt to locate Model Optimizer trying to use PyPI distribution", where both "attempt" and "trying" communicate the same meaning.

And second, it doesn't explain much. What PyPI distribution are you talking about? How is it used to locate Model Optimizer? I would argue that PyPI itself is actually rather irrelevant here.

Reply:
Removed PyPI mention at all. With https://github.com/openvinotoolkit/openvino/pull/4801 both PyPI and and just `export PYTHONPATH=/path/to/openvino/model-optimizer/:$PYTHONPATH` may work

Reply:
That's a lot better, but I think you should still explicitly say that it will try to locate MO by searching for the `mo` package in the Python environment.